src/scip.rs

```diff
@@ -292,6 +292,26 @@ impl ScipPtr {
         Some(unsafe { ffi::SCIPgetBestSol(self.raw) })
     }
 
+    /// Returns a vector containing SCIP_SOL pointers.
+    /// The vector needs to be dropped manually
+    pub(crate) fn get_sols(&self) -> Option<ManuallyDrop<Vec<*mut SCIP_SOL>>> {
+        // check number of sols
+        let n_sols = self.n_sols();
+        if n_sols == 0 {
+            return None;
+        }
+        let sols = unsafe {
+            //? ManuallyDrop is needed to prevent Vec::drop from running.
+            //? This prevents Rust from calling free on the elements in the vector.
+            ManuallyDrop::new(Vec::from_raw_parts(
```

Comment:
Why is calling Vec::drop a problem? I don't think it will free the SCIP solution objects as it only carries a reference to them. Please correct me if I'm wrong here. 
You can do the same logic as in the `conss()` method.